src/main/java/com/fabriccommunity/spookytime/MixinHelpers.java

```diff
@@ -0,0 +1,26 @@
+package com.fabriccommunity.spookytime;
+
+import net.minecraft.entity.EquipmentSlot;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.Items;
+
+import java.util.Random;
+
+/**
+ * @author Indigo Amann
+ */
+public class MixinHelpers {
+    public static final Random random = new Random();
+    public static ItemStack getEquippedPumpkin(LivingEntity entity, EquipmentSlot slot) {
+        if (slot != EquipmentSlot.HEAD) return entity.getEquippedStack(slot);
+        ItemStack stack = entity.getEquippedStack(EquipmentSlot.HEAD);
+        if (stack.isEmpty()) {
+            if (random.nextInt(3) == 0) {
```

Comment:
I think 3 is a _tad_ bit too much. We can definitely add a config option for it later on, but for now, I would say ~30-40 seems more appropriate. Another option would be using 50 as default and moving it to 3 (or something very common) just for the special dimension (which I think works much better).

Reply:
1/5 wan't enough for me xD. I'll change it to 1/10 tho